tensorflow_serving/model_servers/main.cc

```diff
@@ -209,11 +223,20 @@ int main(int argc, char** argv) {
     std::cout << "unknown argument: " << argv[1] << "\n" << usage;
   }
 
+  FileSystemStoragePathSourceConfig_VersionPolicy parsed_version_policy;
+  bool valid_policy = FileSystemStoragePathSourceConfig_VersionPolicy_Parse(
+    model_version_policy, &parsed_version_policy);
+  if (!valid_policy) {
```

Comment:
Instead of the if-statement, please just write:
CHECK(valid_policy) << "Invalid model_version_policy ...